app/src/main/report/components/Versions/ReportVersionSwitcher.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from "react";
+import {VersionIdentifier} from "../../models/VersionIdentifier";
+import {longTimestamp} from "../../../shared/Helpers";
+import {IRouter, Router} from "simple-react-router";
+import {VersionInfoPage} from "./VersionInfoPage";
+
+const styles = require("../../styles/reports.css");
+
+interface Props {
+    currentVersion: string;
+    versions: string[];
+    onChangeVersion: (version: string) => void;
+}
+
+export class ReportVersionSwitcher extends React.Component<Props, undefined> {
+    constructor() {
+        super();
+        this.onSelect = this.onSelect.bind(this);
+    }
+
+    onSelect(e: React.ChangeEvent<HTMLSelectElement>) {
+        this.props.onChangeVersion(e.target.value);
+    }
+
+    render() {
+        const versions = this.props.versions.map(v => new VersionIdentifier(v));
+        versions.sort((a, b) => b.compareTo(a));
+        const items = versions.map(v => {
+            return <option value={v.id} key={v.id}>
+                {longTimestamp(v.timestamp)}
+            </option>
+        });
+        return <div className={"text-right mb-3 " + styles.versionSwitcher}>
+            Report version &nbsp;
+            <select onChange={this.onSelect} value={this.props.currentVersion}>
+                {items}
```

Comment:
Cool, so you went with single line/vanilla html select. The only other place I think we use a select so far is on the download demographics page, where I've styled it with the bootstrap `form-control` class (see https://support.montagu.dide.ic.ac.uk:10443/contribution/Suraratdecha/responsibilities/201708test-2/demographics/). To keep styling consistent, we should either use that class here too, or remove from the demographics page...

Actually the bootstrap class adds a border-radius, which is a little inconsistent with the rest of the site. We should probably move to including Bootstrap's .scss files and compiling them using webpack rather than just including the pre-compiled css. Then we can make proper use of its variables, e.g. setting `$enable-rounded: false` will remove all rounded corners.

Reply:
It turned out to be really complicated just as it was, so I thought I'd do vanilla select here, and we can always improve the look and feel later. I have added Bootstrap styling as discussed.